Holovibes/sources/thread/file_frame_read_worker.cc

```diff
@@ -269,17 +267,18 @@ void FileFrameReadWorker::enqueue_loop(size_t nb_frames_to_enqueue)
             {
```

Comment:
The while condition should be:
`api::get_input_queue()->get_total_nb_frames() + setting<settings::BatchSize>() - 1 >= api::get_input_queue()->get_size() && !stop_requested_`

Since now you are enqueuing a whole batch, before enqueuing it you must be sure that there is enough place for it otherwise there will be dropped frame.

Reply:
Done